exporters/stdout/stdoutlog/record.go

```diff
@@ -0,0 +1,59 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package stdoutlog // import "go.opentelemetry.io/otel/exporters/stdout/stdoutlog"
+
+import (
+	"time"
+
+	"go.opentelemetry.io/otel/log"
+	"go.opentelemetry.io/otel/sdk/instrumentation"
+	sdklog "go.opentelemetry.io/otel/sdk/log"
+	"go.opentelemetry.io/otel/sdk/resource"
+	"go.opentelemetry.io/otel/trace"
+)
+
+// recordJSON is a JSON-serializable representation of a Record.
+type recordJSON struct {
+	Timestamp                 time.Time
+	ObservedTimestamp         time.Time
```

Comment:
Shouldn't these be

```suggestion
	Timestamp                 *time.Time
	ObservedTimestamp         *time.Time
```

To nicely handle `WithoutTimestamps`?

Here is why: https://go.dev/play/p/-a5jI4jOXSK

AFAIK this would be a different behaviour that we have for `stdoutmetric` and `stdouttrace`.

We can handle it via a separate issue and PR as we may have different opinions.

Reply:
Created https://github.com/open-telemetry/opentelemetry-go/issues/5175